products/index.html

```diff
@@ -14,7 +14,14 @@ <h1>Products</h1>
             <div class="medium-4 columns">
                 <h3><img src="{{ site.baseurl }}/img/logos/omero.svg" alt="OMERO logo" class="section-title-img-logo" /></h3>                
                 <h5>Handles your images in a central repo from microscope to publication</h5>
-                <p>You can view, organize, analyze and share your data from anywhere you have internet access. Work with your images from a desktop app (Windows, Mac or Linux), from the web or from 3rd party software.</p>
+                <p>
+                    You can view, organize, analyze and share your data from
+                    anywhere you have internet access. Work with your images
+                    from a desktop app (Windows, Mac or Linux), from the web
+                    or from 3rd party software. <a href="https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/sysadmins/publication.html" target="blank">Publish</a>
```

Comment:
Same here. This seems like something that it would be good to have in:

 * a carousel
 * a randomly changing link
 * a highlight of **new** features
 * a list of all the features

Reply:
We might need to rope @lunson in for that but I can reproduce the suggestion above here too for now

Reply:
I vote for a highlight/list of new features. Ping me when content is ready!